doc/faq.rst

```diff
@@ -67,12 +67,16 @@ to not be included as default messages.
 You can see the plugin you need to explicitly :ref:`load in the technical reference
 <user_guide/checkers/extensions:optional checkers>`.
 
-I want to use pylint on each keystroke in my IDE, how can I do that ?
----------------------------------------------------------------------
+I want to use run pylint automatically in my IDE. How do I do that?
+---------------------------------------------------------------------------
 
-Don't do it: pylint's full suite of checks is not fast enough for that and never
-will be. pylint is best suited for linting on save for small projects, or for a continuous
-integration job or a git ``pre-push`` hook for big projects. The larger your repository
+Pylint will never be fast enough to run on every keystroke.
+
+However, IDEs could run pylint automatically *in the background*
+when the IDE saves changed Python files. Contact your IDE with a support request :-)
```

Comment:
Maybe we should link to the vs code plugin, they've already added a lot of tooling for this.

Reply:
@jacobtylerwalls Good idea. I'll update the PR later today.

Reply:
```suggestion
when the IDE saves changed Python files.
```
Some IDE do it already, some do not and might not like the influx of requests, let's remove the advice.